src/ui/public/notify/lib/_format_msg.js

```diff
@@ -22,6 +24,9 @@ define(function (require) {
       rtn += esMsg;
     } else if (err instanceof Error) {
       rtn += formatMsg.describeError(err);
+    } else if (get(err, 'status') && get(err, 'data')) {
```

Comment:
I think you'll want to use `has` here instead of `get` on the off-chance that angular ever does something funky like return a status of 0 or empty data.